kuma/static/js/wiki-compat-signal.js

```diff
@@ -2,6 +2,669 @@
     'use strict';
 
     var bcTable;
+    var bcSignalBlock;
+    var bcSignalCompleteBlock;
+    var sendReportButton;
+    var nextStepButton;
+    var stepsInfoSpan;
+    var bcSignalStep;
+    var bcSignalSteps = 2;
+
+    /**
+     * Toggles main block visibility and resets form completion progress
+     */
+    function toggleBcSignalBlock () {
+        if (bcSignalCompleteBlock.classList.contains('open')) {
+            bcSignalCompleteBlock.classList.remove('open');
+        } else {
+            bcSignalBlock.classList.toggle('open');
+        }
+
+        resetControls();
+        toStep(1);
+    }
+
+    /**
+     * Moves user to a step, by settings displaying related blocks
+     * @param {Number} step - Step index to be shown
+     */
+    function toStep (step) {
+        if (step !== bcSignalStep) {
+            const prevStep = bcSignalStep;
+            bcSignalStep = step;
+            stepsInfoSpan.innerText = `Step ${bcSignalStep} of ${bcSignalSteps}`;
```

Comment:
```suggestion
            stepsInfoSpan.innerText = 'Step' + ${bcSignalStep} + ' of ' + ${bcSignalSteps};
```

As with `const` for now, stick to "old school" strong concatenation here ;)